src/DynamoMLDataPipeline/DynamoMLDataPipeline.cs

```diff
@@ -269,70 +282,20 @@ static void DataExchangeToForge(string filePath, RestClient client, string token
-            var fileDownloadClient = new RestClient();
-            RestRequest downloadDataRequest = new RestRequest(downloadURL);
-            var downloadDataResponse = fileDownloadClient.ExecuteGet(downloadDataRequest); // Data returned in base 64 compressed
-            var base64Data = downloadDataResponse.Content;
-
-            // STEP 3: Convert base64 compressed data back to .dyn file
-            byte[] compressedData = Convert.FromBase64String(base64Data);
-            byte[] uncompressedData = DataUtilities.Decompress(compressedData);
-            string uncompressedString = Encoding.UTF8.GetString(uncompressedData);
-            string parentDir = Directory.GetParent(filePath).FullName;
-            var dynamoParsedPath = Path.Combine(parentDir, "MaximizeWindowViews-deserialized.dyn");
-            File.WriteAllText(dynamoParsedPath, uncompressedString);
-        }
-
-        public void DataExchange(string filePath)
+        public void SendWorkspaceLog(string filePath)
         {
-            // The Forge team has recommended to use the Stage environment for testing.
-            // Depending on whether we are using Stage or Prod, a Forge token needs to be retrieved 
-            // using client and secret id from a Forge app created in the respective environment.
-            // Assuming there is a way to retrieve the 3-leg Forge token in Dynamo, I will use a hardcoded one here.
+            // Depending on whether we are using Stage or Prod, a token needs to be retrieved 
+            // using client and secret id from the app created in the respective environment.
             var token = GetAuthorizationToken();
 
-            // Stage collectionId created for Dynamo
+            // CollectionId created for Dynamo
             CollectionId = ProductionCollectionID;
 
-            Analytics.TrackEvent(Actions.Export, Categories.DynamoMLDataPipelineOperations, "CollectionID", Convert.ToInt32(CollectionId));
-            //ExchangeContainerId = "";
+            Analytics.TrackEvent(Actions.Export, Categories.DynamoMLDataPipelineOperations, CollectionId);
 
-            var forgeClient = new RestClient(ProductionClientUrl);
+            var client = new RestClient(ProductionClientUrl);
 
-            DataExchangeToForge(filePath, forgeClient, token);
-
-            // This is needed here to allow for the testing workflow to run after the upload has been completed.
-            // It looks like even though the fullfillment ran by teh SendNewLog function is being reported 
-            // as COMPLETED the upload is not done.
-
-            // Test pipeline is being commented out. 
-            //Thread.Sleep(5000);
-            //TestPipeline(filePath, token, forgeClient);
-        }
-
-        private static string GetAuthorizationToken()
-        {
-            return AuthTokenProvider.GetAccessToken();
+            SendToMLDataPipeline(filePath, client, token);
```

Comment:
Just a reminder for the try catch block here.

Reply:
The main function calling this has a try catch, but will add it here as well.

Reply:
Added.